assets/prefabs/MagicFool.prefab

```diff
@@ -1,45 +1,65 @@
-                    }
-                ]
+  "skeletalmesh": {
+    "mesh": "fool",
+    "heightOffset": -0.5,
+    "material": "blackFoolSkin",
+    "animation": "foolStill",
+    "loop": true
+  },
+  "CapsuleShape": {
+    "radius": 1,
+    "height": 2
+  },
+  "persisted": true,
+  "AnimateBounce": {
+    "maxHeight": 0.15,
+    "period": 8.0
+  },
+  "RigidBody": {
+    "kinematic": false,
+    "mass": 0
+  },
+  "Dialog": {
+    "firstPage": "main",
+    "pages": [
+      {
+        "id": "main",
+        "title": "Welcome",
+        "paragraphText": [
+          "Welcome to Light &amp; Shadow, ${player.name}! It is time for you to choose a side in this battle. Approach a teleporter and activate it to join that side in a fight for victory."
+        ],
+        "responses": [
+          {
+            "text": "Very well..",
+            "responseImage": "LightAndShadowResources:answerArrow",
+            "action": {
+              "class": "Dialogs:CloseDialogAction"
             }
+          },
+          {
+            "text": "How does it work?",
+            "responseImage": "LightAndShadowResources:answerArrow",
+            "action": [{
+              "class": "Dialogs:NewDialogAction",
+              "target": "instructions"
+            },
+              {
+                "class": "Dialogs:CloseDialogAction"
+              }
+            ]
+          }
         ]
-    },
+      },
+      {
+        "id": "instructions",
+        "title": "Instructions",
+        "paragraphText": [
+          "As LightAndShadow is a purely multiplayer gameplay now, you'll need a second client to actually start the game.However, you can already take a look at the gameplay arena by interacting (E) with any of the two dice on the jester's platform.When interacting with one of them, you'll join the related red or black team and spawn in the gameplay arena on your team's home base.On your home base, you'll notice a single block, that is the flag. The goal of the game is to fetch the other team's flag from their base and bring it to your home base.You'll also notice there is a magic dome around you that you cannot escape, same goes for the other team's base on the other side of the gameplay arena.As long as there's no player in the other team, these magic domes will not resolve and the game will tell you that it's waiting for more players.As soon as somebody joins the other team (you can try this out locally by starting a second client and joining a locally hosted game), a countdown will start and once that's down to 0, the magic domes will resolve and the game will start.Now it's time to speed to the other team's base to fetch their flag (E) and back to your home base to deliver it (again E). But take care, the players of the other team could try to hinder you. All players have a basic set of weapons to fight their opponents with.The team that first scores 5 points, aka successfully delivers 5 enemy flags to their home base, wins."
```

Comment:
This is only a copy-paste of what I wrote in Discord. Please rephrase this a bit so it sounds more like something the fool would tell a player in-game.

Reply:
I made some changes and rephrased the text